<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GLDRichTextEditFindAndReplaceWidget</class>
 <widget class="QWidget" name="GLDRichTextEditFindAndReplaceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GLDRichTextEditFindAndReplaceWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>查找内容：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtFindText"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnFind">
       <property name="text">
        <string>查找下一个</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labReplace">
       <property name="text">
        <string>替换为：  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtReplaceText"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnReplace">
       <property name="text">
        <string>替换</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="cbxWord">
         <property name="text">
          <string>全字匹配</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbxCase">
         <property name="text">
          <string>区分大小写</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="grpBox">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>方向</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="rbtnUp">
          <property name="text">
           <string>向上</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbtnDown">
          <property name="text">
           <string>向下</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnReplaceAll">
         <property name="text">
          <string>全部替换</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
